tooling/monitoring-dashboard/README.md

```diff
@@ -0,0 +1,34 @@
+# Monitoring Dashboard for WSO2 Micro Integrator
+
+## Building from the source
+
+- ### Setting up the development environment
+1. Install Node.js [8.X.X](https://nodejs.org/en/download/releases/)
+2. Fork the [repository](https://github.com/wso2/micro-integrator)
+3. Clone your fork into any directory
+4. `cd` into cloned directory and then cd into `micro-integrator/tooling/monitoring-dashboard`.
```

Comment:
then `cd` into...
This directory will be reffered to as, <DASHBOARD_HOME> here onwards.